apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Snippets/UrlSignerSnippets.cs

```diff
@@ -152,18 +152,21 @@ public async Task SignedURLPut()
         internal sealed class IamServiceBlobSigner : UrlSigner.IBlobSigner
         {
             private readonly IamService _iamService;
+
             public string Id { get; }
 
+            public string Algorithm => "GOOG4-RSA-SHA256";
```

Comment:
IIUC, @amanda-tarafa this is an initial PR; for Snippets; does a dev need to supply the Algorithm for C# or is this determined by the Signer based on the selected implementation?

Reply:
@frankyn this does not add support for HMAC yet. I've only made the breaking changes that we needed to add support for HMAC later on. Making the breaking changes was the most urgennt bit re the bumping of the major version. We'll add support for HMAC after we have bumped the the libraries. I'll make sure to link you on that PR.